app/controllers/consolidations/MovementConfirmationController.scala

```diff
@@ -14,12 +14,11 @@
  * limitations under the License.
  */
 
-package controllers
+package controllers.consolidations
```

Comment:
So is this for all journeys now because I understood cosolidations to just be the Associate/DIssociate/SHut journeys?